source/expressions.tex

```diff
@@ -1388,7 +1422,20 @@
 \end{note}
 
 \pnum
-The result is the entity denoted by the identifier.
+A \defnadj{component}{name} of an \grammarterm{unqualified-id} $U$ is $U$
+if it is a name or the component name of the \grammarterm{template-id} or \grammarterm{type-name} of $U$, if any.
```

Comment:
"component name" is one of those definitions that is distributed across the standard, because it is closely related to the grammar structure (which is introduced later for template-id and other things).
Same for "attribute appertains to", for example.

I'll add bullets; your bullets cause the wrong grouping.